src/GafferScene/CopyPrimitiveVariables.cpp

```diff
@@ -212,5 +227,5 @@ bool CopyPrimitiveVariables::adjustBounds() const
 		return false;
 	}
 
-	return StringAlgo::matchMultiple( "P", primitiveVariablesPlug()->getValue() );
+	return StringAlgo::matchMultiple( "P", primitiveVariablesPlug()->getValue() ) && prefixPlug()->isSetToDefault();
```

Comment:
This is pretty evil, but it looks like currently I can name a variable "" using OSLCode.  So I could then write to P by using the prefix "P", and this would theoretically require a bounds adjustment.  Solution is probably that we should reject "" as a variable name.

Reply:
I imagine a primitive variable called "" would cause all sorts of other problems, so I'm inclined to merge this as-is and deal with that separately. I've pushed a rebase to fix the merge conflict.